dvc/repo/brancher.py

```diff
@@ -30,10 +30,10 @@ def brancher(  # noqa: E302
     scm = self.scm
 
     self.tree = WorkingTree(self.root_dir)
-    yield "working tree"
+    yield "workspace"
 
-    if revs and "working tree" in revs:
-        revs.remove("working tree")
+    if revs and "workspace" in revs:
```

Comment:
There actually can be a tag or branch named `workspace` so this is no longer safe.

Reply:
This will also clash in many other places like plots and metrics.

Reply:
@Suor, is there any need for `workspace in revs` check at all?

Reply:
Making it a flash won't fix output.